regression/pages/studio/asset_index_studio.py

```diff
@@ -111,3 +104,17 @@ def sort_assets(self):
         Sort the assets
         """
         click_css_with_animation_enabled(self, '.column-sort-link', 0, False)
+
+    def get_page_count(self):
```

Comment:
It returns page count as string. Either you can return a numerical value or can change the docstring. I know what you are trying to say, but its little ambiguous at the moment.